README.md

```diff
@@ -20,8 +20,14 @@ docker compose build
 docker compose up
 ```
 
-See the [Quick Start](documentation/documentation.md#quick-start) section in the documentation for a complete example of how to initialize the plugin and store events and queries. 
+See the [Quick Start](documentation/documentation.md#quick-start) section in the documentation for a complete example of how to initialize the plugin and store events and queries.
 
 ## Releasing
-
 Run `tag-and-release.sh` to create and push a tag to GitHub. The tag will kick off the GitHub Action to build the release. After running `tag-and-release.sh`, update the version numbers in `gradle.properties` and the `Dockerfile` and commit the changes.
+
+## License
+This code is licensed under the Apache 2.0 License. See [LICENSE.txt](LICENSE.txt).
+
+## Copyright
+Copyright OpenSearch Contributors. See [NOTICE.txt](NOTICE.txt) for details.
```

Comment:
@epugh IANAL so see if this is acceptable.

Reply:
I think so, for now.  If for some reason this doesn't move forward we would reevaluate.